projects/pastanaga/src/lib/textfield/input.component.html

```diff
@@ -11,7 +11,7 @@
            [attr.placeholder]="placeholder | translate"
            [value]="!!value || value === 0 ? value : ''"
            [readonly]="isReadOnly"
-           [disabled]="disabled"
+           [disabled]="disabled? '': null"
```

Comment:
why not `[disabled]="disabled ? disabled : null"`
formatting is wrong (missing space)

Reply:
Actually, try with the fix I gave in the component ts, and here use the protected value (make it public if needed) instead of the getter: `[disabled]="_disabled"`